db/schema.sql

```diff
@@ -0,0 +1,92 @@
+-- TODO(brandon): check if deployed version of PostgreSQL has gen_random_uuid built in and remove pgcrypto extension if so
```

Comment:
PostgreSQL versions 13 and 14 support gen_random_uuid, and Cloud SQL for PostgreSQL supports both versions (plus several older ones)

Reply:
Acknowledged--the version of postgres included in my workstation's package manager is too old (12.9). We may want to keep this for "backwards compatibility" until everyone is on a recent-enough version.